integration/integration_test.go

```diff
@@ -1218,10 +1219,20 @@ func runDisconnectTest(t *testing.T, suite *integrationTestSuite, tc disconnectT
 				return
 			default:
 			}
+
+			isSSHError := func(e error) bool {
+				switch trace.Unwrap(err).(type) {
+				case *ssh.ExitError, *ssh.ExitMissingError:
+					return true
+				default:
+					return false
+				}
+			}
```

Comment:
Could you extract this into a standalone function `isSSHError` (within this file) so that it can be reused in `testPAM` and `testWindowChange` where virtually the same `switch` is introduced again?